src/test/scala/edu/ie3/simona/model/participant/BMModelSpec.scala

```diff
@@ -32,40 +30,19 @@ import java.util.UUID
 class BMModelSpec extends UnitSpec {
 
   implicit val powerTolerance: Power = Megawatts(1e-4)
```

Comment:
Sorry, should have been more precise. I would prefer a lower tolerance. Let's try Kilowats(1e-6) or at least e-4...